app/queries/ects/with_an_appropriate_body_and_unregistered_query.rb

```diff
@@ -6,8 +6,9 @@ def call
       InductionRecord
         .active
         .training_status_active
-        .joins(:participant_profile)
+        .joins(participant_profile: :ecf_participant_eligibility)
         .where(participant_profile: { induction_start_date: nil, type: "ParticipantProfile::ECT" })
+        .where(ecf_participant_eligibility: { qts: true, active_flags: false, different_trn: false, previous_induction: false, no_induction: true })
```

Comment:
Do these conditions apply anywhere else too? Worth combining them into a scope?

Reply:
Good question. It might make it clearer if these were in a scope/scopes.